File: Helpers/FileHelper.cs
using AdventOfCode2023.Constants;

namespace AdventOfCode2023.Helpers
{
    public static class FileHelper
    {
        public static string[] GetLines(int day, bool isSample = false)
        {
            string suffix = isSample ? "sample" : "input";
            var filePath = $"{AdventConstants.FolderPath}day{day}_{suffix}.txt";

            Console.WriteLine($"Read file {filePath}");
            return File.ReadAllLines(filePath);
        }
    }
}

File: ts/helpers/file.helper.ts
import * as fs from "fs";

export function getLines(day: number, isSample: boolean) {
    const suffix: string = isSample ? 'sample': 'input';
    const path: string = `inputs/day${day}_${suffix}.txt`;
    return getLinesFromPath(path);
}

export function getLinesFromPath(path: string) {
    console.log(`Get lines from ${path}`);
    return fs.readFileSync(path, {encoding: "utf-8"}).split('\n');
}
